src/components/Sidebar/Sidebar.js

```diff
@@ -133,6 +120,7 @@ const buttons = [
   {
     icon: 'book',
     text: 'Narrative Mode',
+    link: '/narrative',
```

Comment:
I think better than adding a link to this one, is removing it since it's not a part of V1. We should probably have these links (in my opinion in this order):

`/characters`
`/charms`
`/qualities`
`/equipment`
`/combat`